foreground_extraction.py

```diff
@@ -116,3 +117,38 @@ def get_foreground_mask_hog(self, frames):
             fgmask = np.where((fgmask == 2) | (fgmask == 0), 0, 1).astype('uint8')
             fgmasks.append(fgmask)
         return np.array(fgmasks)
+        
+    def get_foreground_mask_dof(self, frames):
```

Comment:
remove unused/commented-out code

Reply:
which lines are you saying?

Reply:
all commented-out lines should be removed because those lines are only for testing and should only exist in local